extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -40,6 +47,14 @@ public function renderInfobox( $text, $params, $parser, $frame ) {
 		return [ $renderedValue, 'markerType' => 'nowiki' ];
 	}
 
+	private function renderUnimplementedTagErrorMesssage( $tagName ) {
+		$renderedValue = '<strong class="error"> '
+			. wfMessage( 'unimplemented-infobox-tag' ) . ': '
```

Comment:
You should pass $tagName as a message param as depending on the language the order of words may be different